standards/countries.rst

```diff
@@ -0,0 +1,44 @@
+.. _countries:
+
+A common list of countries
+==========================
+
+Having a standardized list of country names is an important prerequisite for region
+definitions, scenario analysis and model comparison.
+
+The :class:`nomenclature` package (`read the docs`_) builds on the :class:`pycountry`
+package to provide a standardized list if country names based on the
+`ISO 3166-1 standard`_.
+
+For consistency with established conventions in the modelling community, several country
```

Comment:
I would list them all instead of mentioning just one and leaving the other ones ambiguous.

Reply:
Disagree, this is bound to lead to mistakes at some point because we forget to keep this in sync. An alternative would be to build this on the fly from the python mapping, but that's beyond my capacity at the moment.